faststream/kafka/broker.py

```diff
@@ -368,6 +378,7 @@ def subscriber(  # type: ignore[override]
             max_poll_records=max_poll_records,
             exclude_internal_topics=exclude_internal_topics,
             isolation_level=isolation_level,
+            **parse_security(self.security),
```

Comment:
You need to remove these (security) arguments (and add your security) from
* kafka/broker.pyi __init__, connect
* kafka/shared/schemas.py
* kafka/fastapi.pyi __init__

Reply:
Also, remove these line from here: Producer and Consumer shared arguments are passing to Consumer connection in the 179 and 199 lines

Reply:
Probably, you don't need to patch `kafka/shared/schemas.py`, but unpuck `parse_security` in the `_connect` method in the 179 line

Reply:
> You need to remove these (security) arguments (and add your security) from
> 
> * kafka/broker.pyi **init**, connect
> * kafka/shared/schemas.py
> * kafka/fastapi.pyi **init**

- Updated signatures, schema left the same 
- Unpacking of security params is moved to _connect method